ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/ProcessValidation.scala

```diff
@@ -154,15 +154,10 @@ class ProcessValidation(validators: ProcessingTypeDataProvider[ProcessValidator]
   }
 
   private def validateDuplicates(displayable: DisplayableProcess): ValidationResult = {
-    val groupIds = displayable.metaData.additionalFields
-      .flatMap(a => a.cast[ProcessAdditionalFields])
-      .toList
-      .flatMap(_.groups)
-      .map(_.id)
     val nodeIds = displayable.nodes.map(_.id)
 
     //in theory it would be possible to have group named like one of nodes inside, but it's not worth complicating logic...
```

Comment:
hmm it this comment still valid ? even more, can we have duplicates right now ?

Reply:
:+1: comment is not valid, but we can have duplicates defined in FE